src/batch-client/index.ts

```diff
@@ -306,23 +306,34 @@ export class ZimbraBatchClient {
 		const { attributes, ...rest } = data;
 		const contactAttrs = <Object[]>[];
 
-		forEach(attributes, (val, key) =>
-			contactAttrs.push({
-				name: key,
-				[key === 'image' ? 'aid' : 'content']: val
-			})
+		forEach(
+			attributes,
+			(val, key) =>
+				key !== 'other' &&
+				contactAttrs.push({
+					name: key,
+					[key === 'image' ? 'aid' : 'content']: val
+				})
 		);
-
+		let body = {
+			cn: {
+				...denormalize(ContactInputRequest)({
+					...rest,
+					attributes: contactAttrs
+				})
+			}
+		};
+		attributes &&
```

Comment:
instead of multiple &&, can we use lodash.get() ?

Reply:
Removed. Changed logic.